src/main/java/pvpmode/api/server/log/AbstractFileCombatLogHandler.java

```diff
@@ -76,6 +83,20 @@ protected Path getUnusedFileNameWithIndex (Path basicFileNameWithoutFileEnding,
         return result;
     }
 
+    protected String getUsername (UUID uuid)
+    {
+        PlayerProfileCache cache = MinecraftServer.getServer ().func_152358_ax ();
+
+        EntityPlayerMP player = PvPServerUtils.getPlayer (uuid);
+
+        if (player != null)
+            return player.getDisplayName ();
+
+        GameProfile profile = cache.func_152652_a (uuid);
```

Comment:
This variable should be inlined since it only has one usage
```suggestion
        EntityPlayerMP player = PvPServerUtils.getPlayer (uuid);

        if (player != null)
            return player.getDisplayName ();

        GameProfile profile = MinecraftServer.getServer ().func_152358_ax ()..func_152652_a (uuid);
```